theme/static/js/hs_file_browser.js

```diff
@@ -1471,16 +1518,18 @@ $(document).ready(function () {
         pathLog.push(upPath);
         pathLogIndex = pathLog.length - 1;
         get_irods_folder_struct_ajax_submit(resID, upPath);
+        sessionStorage.currentBrowsepath = upPath;
     });
 
     // Move back
     $("#fb-move-back").click(function () {
-        if (pathLogIndex > 0) {
+        if (pathLogIndex >= 2) {
```

Comment:
I'm not following this change... maybe I just need to look at it in the morning again.

Reply:
Though I am not fully sure of the back button functionality, I was having issue with this button. The logic for turning on/off this button is as follows based on my above change:

When the current browsing path is 'data/contents' (root) the value of pathLogIndex is 1. That means we need to move back only if the pathLogIndex > 1. If pathLogIndex is 1 we need to disable this button as we are already at the root.